app/assets/stylesheets/application.css.scss

```diff
@@ -28,3 +28,18 @@
 .downvote {
   color: red;
 }
+
+#foto, #foto img {
+    background: #1ABC9C;
+    border-radius:50%;
+}
+#foto {
+    border-radius: 50%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border, border-radius, height, width